grafonnet/gauge.libsonnet

```diff
@@ -20,20 +30,51 @@
       [if span != null then 'span']: span,
       title: title,
       type: 'gauge',
+      [if pluginVersion != null then 'pluginVersion']: pluginVersion,
       datasource: datasource,
       options: {
         fieldOptions: {
           calcs: [
             calc,
           ],
+          defaults: {
+            mappings: [],
+            max: max,
+            min: min,
+            thresholds: {
```

Comment:
This syntax appears to be new within Grafana 6.7.X, or at the very least since Grafana 6.4.3, where `thresholds` is just an array.

Reply:
I based on Grafana 6.6.0, where gauge defaults look like this
```
"options": {
  "fieldOptions": {
    "calcs": [
      "mean"
    ],
    "defaults": {
      "mappings": [],
      "thresholds": {
        "mode": "absolute",
        "steps": [
          {
            "color": "green",
            "value": null
          },
          {
            "color": "red",
            "value": 80
          }
        ]
      }
    },
    "overrides": [],
    "values": false
  },
  "orientation": "auto",
  "showThresholdLabels": false,
  "showThresholdMarkers": true
},
"pluginVersion": "6.6.0",
```

I don't know what's the best way to provide appropriate gauge, especially since this plugin is very prone to structure changes like renaming or changing object structure.